u2f-ref-code/java/src/com/google/u2f/server/impl/attestation/android/AuthorizationList.java

```diff
@@ -107,8 +150,26 @@ public Builder setAlgorithm(Algorithm algorithm) {
       return this;
     }
 
+    public Builder setKeySize(Integer keySize) {
+      this.keySize = keySize;
+      return this;
+    }
+
+    public Builder setBlockMode(List<BlockMode> blockMode) {
+      this.blockMode = new ArrayList<BlockMode>(blockMode);
```

Comment:
if (blockMode != null) {
  this.blockMode = new ArrayList<BlockMode>(blockMode);
}